kingpin/bin/deploy.py

```diff
@@ -120,7 +120,7 @@ def main():
     if args.build_only:
         try:
             get_main_actor(dry=False)
-        except actor_exceptions.ActorException as e:
+        except Exception as e:
```

Comment:
It still crashes the app ... I explicitly log it and exit with status 1. The problem is that if python exists on its own - then it's status 0, and that's bad.